SparkConnector/sparkconnector/configuration.py

```diff
@@ -1,5 +1,5 @@
 import os, subprocess, shutil, sys, uuid, time, base64, tempfile
```

Comment:
The message of commit sparkconnector: Preserve the default value of extraJavaOptions  has typos, please fix.